server/sonar-web/src/main/js/app/components/nav/settings/SettingsNav.tsx

```diff
@@ -1,240 +0,0 @@
-          </li>
-          <li>
-            <IndexLink to="/admin/settings/encryption" activeClassName="active">
-              {translate('property.category.security.encryption')}
-            </IndexLink>
-          </li>
-          <li>
-            <IndexLink to="/admin/custom_metrics" activeClassName="active">
-              {translate('custom_metrics.page')}
-            </IndexLink>
-          </li>
-          {extensionsWithoutSupport.map(this.renderExtension)}
-        </ul>
-      </li>
-    );
-  }
-
-  renderProjectsTab() {
-    const { customOrganizations } = this.props;
-    const projectsClassName = classNames('dropdown-toggle', { active: this.isProjectsActive() });
-    return (
-      <li className="dropdown">
-        <a className={projectsClassName} data-toggle="dropdown" href="#">
-          {translate('sidebar.projects')} <i className="icon-dropdown" />
-        </a>
-        <ul className="dropdown-menu">
-          {!customOrganizations && (
-            <li>
-              <IndexLink to="/admin/projects_management" activeClassName="active">
-                {translate('management')}
-              </IndexLink>
-            </li>
-          )}
-          <li>
-            <IndexLink to="/admin/background_tasks" activeClassName="active">
-              {translate('background_tasks.page')}
-            </IndexLink>
-          </li>
-        </ul>
-      </li>
-    );
-  }
-
-  renderSecurityTab() {
-    const { customOrganizations } = this.props;
-    const securityClassName = classNames('dropdown-toggle', { active: this.isSecurityActive() });
-    return (
-      <li className="dropdown">
-        <a className={securityClassName} data-toggle="dropdown" href="#">
-          {translate('sidebar.security')} <i className="icon-dropdown" />
-        </a>
-        <ul className="dropdown-menu">
-          <li>
-            <IndexLink to="/admin/users" activeClassName="active">
-              {translate('users.page')}
-            </IndexLink>
-          </li>
-          {!customOrganizations && (
-            <li>
-              <IndexLink to="/admin/groups" activeClassName="active">
-                {translate('user_groups.page')}
-              </IndexLink>
-            </li>
-          )}
-          {!customOrganizations && (
-            <li>
-              <IndexLink to="/admin/permissions" activeClassName="active">
-                {translate('global_permissions.page')}
-              </IndexLink>
-            </li>
-          )}
-          {!customOrganizations && (
-            <li>
-              <IndexLink to="/admin/permission_templates" activeClassName="active">
-                {translate('permission_templates')}
-              </IndexLink>
-            </li>
-          )}
-        </ul>
-      </li>
-    );
-  }
-
-  render() {
-    const { editionStatus, extensions } = this.props;
-    const hasSupportExtension = extensions.find(extension => extension.key === 'license/support');
-
-    let notifComponent;
```

Comment:
By removing that settings navbar we loose the notifications system of the marketplace.
So these notifications should probably be moved to the default organization navbar and be visible only for users with admin (root) rights.